v21.1/drop-region.md

```diff
@@ -173,6 +173,46 @@ SHOW REGIONS FROM DATABASE foo;
 (0 rows)
 ~~~
 
+You cannot drop a region from a database if the databases uses [`ZONE` survival goal](multiregion-overview.html#surviving-region-failures) and there are only three regions configured on the database:
```

Comment:
Did you mean to say `ZONE` survival goal and link to the **Surviving region failures** section?